LocalDynamoDb/LocalDynamo.cs

```diff
@@ -1,31 +1,30 @@
 ﻿using System;
-using System.ComponentModel;
 using System.Diagnostics;
 using System.IO;
 using System.Reflection;
+using System.Threading;
 using Amazon.DynamoDBv2;
 using Amazon.Runtime;
 
 namespace LocalDynamoDb
 {
     public class LocalDynamo
     {
-        private int _port;
+        private readonly int _port;
         private Process Dynamo { get; set; }
         public AmazonDynamoDBClient Client { get; private set; }
 
-
         public LocalDynamo(int portNumber = 8000)
         {
             _port = portNumber;
             Dynamo = Create(_port);
             Client = CreateClient();
         }
 
-        private Process Create(int portNumber)
+        private static Process Create(int portNumber)
         {
             var processJar = new Process();
-            var arguments = String.Format("-Djava.library.path=./DynamoDBLocal_lib -jar DynamoDBLocal.jar -sharedDb -inMemory -port {0}", portNumber);
+            var arguments = $"-Djava.library.path=./DynamoDBLocal_lib -jar DynamoDBLocal.jar -sharedDb -inMemory -port {portNumber}";
```

Comment:
At some point in the future this location should be maybe configurable, rather than hard-coded to the current directory.

Reply:
Agreed. Would also like to add the ability to launch a Docker container from this too, but figure that'll be another PR.